app/src/main/java/com/babylon/wallet/android/data/transaction/DappRequestFailure.kt

```diff
@@ -39,6 +39,8 @@ sealed class DappRequestFailure(msg: String? = null) : Exception(msg.orEmpty())
         data class FailedToPollTXStatus(val txId: String) : TransactionApprovalFailure()
         data class GatewayRejected(val txId: String) : TransactionApprovalFailure()
```

Comment:
since we have `PermanentlyRejected` and `TemporarilyRejected` do we also need the `GatewayRejected`?
is it used anywhere?

Reply:
> GatewayRejected

Yes, it seems like it can be removed.